deeplake/core/chunk_engine.py

```diff
@@ -764,7 +769,7 @@ def _samples_to_chunks(
                 pbar.update(num_samples_added)
         if progressbar:
             pbar.close()
-
+        self.start_chunk = current_chunk  # type: ignore
```

Comment:
this can cause issues, samples_to_chunks is also called during rechunking, in which case current chunk wouldn't be start chunk for future extends